src/action-ros-ci.ts

```diff
@@ -467,7 +467,9 @@ done`;
 			`--packages-up-to ${packageNames}`,
 			`${extra_options.join(" ")}`,
 			extraCmakeArgs !== "" ? `--cmake-args ${extraCmakeArgs}` : "",
-		].join(" ");
+		]
+			.filter((e) => e.length > 0)
```

Comment:
these are all fine, but now that we are doing it 4 times, i wonder if a utility function would be warranted, e.g.

```
function filterNonEmptyJoin(values: List) {
  return values.filter((v) => v.length > 0).join(" ")
}
```

Reply:
sure I can do that!

I can even add a test :grin:

Reply:
> utility function

done in 1ed8912b4517b05eb6c4959f3619f81bfebed470 with test in 5dde110328c46bf19d83027f833378aa45f71921